addOns/selenium/src/main/java/org/zaproxy/zap/extension/selenium/internal/FirefoxProfileManager.java

```diff
@@ -65,8 +66,12 @@ private Path getProfilesDirectory() {
                         System.getProperty("os.name"));
             }
         }
-        if (!profileDirectory.toFile().isDirectory()) {
+        if (profileDirectory != null && !Files.isDirectory(profileDirectory)) {
+            LOGGER.debug(
+                    "The profiles path is not a directory or does not exist: {}", profileDirectory);
```

Comment:
Is it worth checking writable? Or parent writable?

Reply:
I'm not changing the existing (working) logic.